### ISSUE TITLE
CRITICAL - not possible to login or register to learn and pass certifications 

### ISSUE THREAD

**Issue by UintraContact** [MicrosoftDocs/feedback#1122]
Hi, i am really stuck here https://techprofile.microsoft.com/en-US/register and cant login with my existing account, neither register new. it always says, "You are already registered." regardless whether i try to login or register!

this is already 3 times i have duplicated the same critical issue + sent 4 emails to learn-microsoft support contact. just curious, what should i do more. to be able to learn how loged on or registered (whatever) user?  

thanks in advance for your help!

**Comment by ibakirov** [comment]
Try to log out. Also check by starting new Incognito window in your browser and log in.

**Comment by UintraContact** [comment]
> Try to log out. Also check by starting new Incognito window in your browser and log in.

hi @ibakirov , thanks for tips, unfortunately, same issue here in private / incognito window chrome / mozilla borwsers https://www.screencast.com/t/9IC6a45h 

is that working for you?

**Comment by ibakirov** [comment]
The register form has some errors. Whatever you name give it outputs: Something get wrong as in picture  below

![screenshot_2019-02-07 register technical profile](https://user-images.githubusercontent.com/7983476/52401482-f05fc480-2aeb-11e9-9536-f46baf9d73dc.png)

As you got message that Already registered, you may freely use Sign In button above to log in that account. If it is not registered, also you can register via Sign In above link and create new account or reset password.

**Comment by ibakirov** [comment]
@UintraContact for my account profile: ibakirov works without errors. For your: https://techprofile.microsoft.com/en-us/againandagainvadym goes to 404 error

**Comment by ThomsonPC** [comment]
I am getting the exact same error. I have attempted numerous Usernames and various browsers. all with same error message

**Comment by Paul-Hadfield** [comment]
Yep, looks like this is still ongoing.  Can't proceed, Chrome doesn't give any error message, firefox/edge show the something went wrong....

**Comment by rootsmusic** [comment]
Report feedback at https://docs.microsoft.com/en-us/learn/support/troubleshooting#report-feedback
